### ISSUE TITLE
Limited number of slides

### ISSUE THREAD

**Issue by KingJino** [fabiorino/crelly-slider#65]
Everytime I try to add more than 55 slides I get an error, there's a way to prevent this error and continue to adding slides?

**Comment by fabiorino** [comment]
Hi, not sure why it's happening, there shouldn't be a limitation to the number of slides. Does it matter if the slides contain elements or not?

**Comment by KingJino** [comment]
Nope, if I create a 56th slide with no elements in it there is still an error when I try to save it.
I tried to switch browser, I usually use Firefox and I tried with Chrome but the error persist.

**Comment by fabiorino** [comment]
I'm sorry for the giant delay in the response, I had to dedicate a lot of time into other things lately... I'm wondering if, for some, reason, the server sets a hard limit to 55 values to insert. Do you have access to the SQL logs?
